src/shared/components/adminTabs/Teams/TeamsTab.tsx

```diff
@@ -95,7 +104,7 @@ const Teams = () => {
             label="Punctele echipei 1"
             variant="standard"
             value={team1Points}
-            onChange={(e) => setTeam1Points(+e.target.value)}
+            onChange={(e) => setTeam1Points(e.target.value)}
```

Comment:
Here let's also check if the input is a number, as specified in one of the previous comments